src/assets/css/App.css

```diff
@@ -0,0 +1,62 @@
+#root {
```

Comment:
I would recommend you use css modules and create a  file with all the setup that you need, like fonts, colors , etc across the project. I think they are supported by default, it also helps to avoid collisions among classes as it scoped the classes to the component itself